hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -164,10 +164,13 @@
   public static final String HDDS_X509_FILE_NAME = "hdds.x509.file.name";
   public static final String HDDS_X509_FILE_NAME_DEFAULT = "certificate.crt";
 
+  public static final String HDDS_X509_CRL_NAME = "hdds.x509.CRL.name";
```

Comment:
@abhishekaypurohit @anuengineer Sorry for flagging this too late: config properties should be also added to `ozone-default.xml` (so that it can be exposed to users), or as an exception in `TestOzoneConfigurationFields#addPropertiesNotInXml` (if it's for developers only).  We have an integration test to catch such config mismatches (`TestOzoneConfigurationFields`), but it's currently disabled with all other integration tests...

Filed https://issues.apache.org/jira/browse/HDDS-2873 for this new property.  Please take a look.